rdkit/__init__.py

```diff
@@ -3,3 +3,13 @@
 except ImportError:
   __version__ = 'Unknown'
   raise
+
+# if we are running in a jupyter notebook, enable the extensions
+try:
+  get_ipython()
```

Comment:
I think the idea here is really good, but `get_ipython()` also returns something if we are running in an ipython shell.
Maybe it's possible to detect whether or not we're running in jupyter?

Reply:
Should work now.